### ISSUE TITLE
DisplayGTFSStops error import | 'stop_id' is not in list

### ISSUE THREAD

**Issue by laurentwww** [Esri/public-transit-tools#115]
Hi,

When executing the script to import the stops.txt GTFS file, I got an error as below. All files in local disk :

Exécution du script DisplayGTFSStops...
Reading input stops.txt file...
 Warning! Your stops.txt file does not contain the required %s field. This tool will run correctly anyway, but your GTFS file is invalid.
Error validating stops.txt file fields.
Failed to create a feature class of GTFS stops.
Failed script DisplayGTFSStops...
Traceback (most recent call last):
  File "C:\ArcGIS_Data\Tools\DisplayGTFSInArcGIS_2.0.4\scripts\CreateStopsFC.py", line 110, in <module>
    stop_id_idx = columns.index("stop_id")
ValueError: 'stop_id' is not in list
Échec de l’exécution de (DisplayGTFSStops).

Sample of stops.txt file :
stop_id,stop_name,stop_lat,stop_lon,location_type,parent_station
"1101064","Malpensa Aeroporto, terminal 1","45.6272144319203","8.7111160249179","",""
"1101069","Anzola, chiesa","45.9899010293845","8.34506152974108","",""
"1322000","Altoggio","46.1672513851495","8.345807131427","",""
"1322001","Antronapiana","46.060121674738","8.11361957990831","",""

**Comment by mmorang** [comment]
Hello.  Could you attach your stops.txt file, or send me a link where I can download it?  For some reason my script thinks that the sotp_id field does not exist, and I'd like to figure out why.

**Comment by mmorang** [comment]
Are you using ArcGIS Pro?

**Comment by laurentwww** [comment]
Yes, ArcGIS Pro 2.0.0

[stops.txt](https://github.com/Esri/public-transit-tools/files/1803575/stops.txt)
[error.txt](https://github.com/Esri/public-transit-tools/files/1803576/error.txt)

**Comment by mmorang** [comment]
I have fixed the problem.  Your data has a BOM (bit order mark), and although I had made changes to my other scripts to correctly handle this case, somehow I had overlooked this one.  It should work if you get the latest version from this repo or download the toolbox from http://arcg.is/1iuez8.

I also fixed the problem with the error message.  It should fail out properly if someone is actually missing the "stop_id" field.
